safeweb/http.go

```diff
@@ -195,6 +196,20 @@ func NewServer(config Config) (*Server, error) {
 	return s, nil
 }
 
+// getMoreSpecificPattern returns either a or b, depending on which pattern
+// contains more pathname components (i.e. is more specific). If they are
+// equally specific, returns an empty string.
+func getMoreSpecificPattern(a, b string) string {
```

Comment:
optional, but the Java-hater in me does not love getter-style funcs
```suggestion
// moreSpecificPattern returns either a or b, depending on which pattern
// contains more pathname components (i.e. is more specific). If they are
// equally specific, returns an empty string.
func moreSpecificPattern(a, b string) string {
```

Reply:
+1

[Unnecessary context: All functions that return a value "get" something, so mainly the prefix is useful when there's a namespace conflict. That said—it triggers me less here because unexported. Mostly to be avoided in exported APIs].